static/app/views/insights/database/views/databaseSpanSummaryPage.tsx

```diff
@@ -228,6 +252,20 @@ export function DatabaseSpanSummaryPage({params}: Props) {
               </DescriptionContainer>
             )}
 
+            <Feature features="insights-related-issues-table">
+              {!areIndexedSpansByGroupIdLoading && (
+                <ModuleLayout.Full>
+                  <InsightIssuesList
+                    issueTypes={[
+                      'performance_slow_db_query',
+                      'performance_n_plus_one_db_queries',
+                    ]}
+                    message={indexedSpansByGroupId[0]['span.description']}
```

Comment:
We _should_ get an indexed span here, but we should probably protect against the case where we don't.
```suggestion
                    message={indexedSpansByGroupId[0]?.['span.description']}
```